lib/govuk_admin_template/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukAdminTemplate
-  VERSION = "6.8.0".freeze
+  VERSION = "6.9.0".freeze
```

Comment:
Is that actually a bug fix? and thus should be 6.8.1 ?

Reply:
Yea good point! I have checked the fixes locally. Happy to deploy the branch out and check on int though